### PR TITLE
Add better docker-compose example

### PR DESCRIPTION
**Pull request by ThomDietrich**
Hey!
Went a bit back and forth on this. My intention was to make the use of excludes as useful as possible, for me and other users. I started by adding a new env but realized that the right use of multiline yaml is all that is needed.

The change highlights the right use of multiline for backup args and gives a better forget example.

The PR also introduces BACKUP_ARGS as a way to pass tags. Imho this is the better way to offer this option in the long run. For now no real changes in this regard - if you are willing to consider it I would suggest to gradually deprecate `RESTIC_BACKUP_TAGS` in favor of the more versatile `RESTIC_FORGET_ARGS`.

Best! th

### CODE REVIEW

**Comment by djmaze** [comment]
>  if you are willing to consider it I would suggest to gradually deprecate RESTIC_BACKUP_TAGS in favor of the more versatile RESTIC_FORGET_ARGS.

You mean *in favor of the more versatile RESTIC_BACKUP_ARGS*? I support that, let's do it. Maybe remove RESTIC_BACKUP_TAGS from the example and mark it as deprecated in the documentation.

:+1: for the other changes.

File: README.md [hunk 1]
```diff
@@ -62,15 +62,14 @@ E.g.
...
+* `BACKUP_CRON` - A cron expression for when to run the backup. E.g. `0 30 3 * * *` in order to run every night at 3:30 am. See [the go-cron documentation](https://godoc.org/github.com/robfig/cron) for details on the expression format
 * `PRUNE_CRON` - A cron expression for when to run the prune job. E.g. `0 0 4 * * *` in order to run every night at 4:00 am. See [the go-cron documentation](https://godoc.org/github.com/robfig/cron) for details on the expression format
 * `RESTIC_REPOSITORY` - location of the restic repository. You can use [any target supported by restic](https://restic.readthedocs.io/en/stable/030_preparing_a_new_repo.html). Default `/mnt/restic`
 * `RESTIC_BACKUP_SOURCES` - source directory to backup. Make sure to mount this into the container as a volume (see the example configs). Default `/data`
 * `RESTIC_PASSWORD` - password for the restic repository. Will also be used to initialize the repository if it is not yet initialized
-* `RESTIC_BACKUP_ARGS` - Optional. Additional arguments given to *restic backup*.
-* `RESTIC_BACKUP_TAGS` - Optional. Tags to set on each snapshot, separated by commas. E.g. `swarm,docker-volumes`
```
Comment (djmaze) [inline README.md:71]: I think as long as we still support this in the code, we should leave `RESTIC_BACKUP_TAGS` in the documentation as well. Just mark it as deprecated here and point to `RESTIC_BACKUP_ARGS` instead.

Comment (ThomDietrich) [reply README.md:71]: will do

**Comment by djmaze** [comment]
That looks good! Just a small conflict to resolve now.

**Comment by ThomDietrich** [comment]
That's all. Nice progress. We will use restic more extensively from now on. Look out for further PRs ;)

**Comment by djmaze** [comment]
Thanks for your help! Appreciated :)

**Pull request merged**